### PR TITLE
WIP - Coach Tools' Actions & Store

### PR DESCRIPTION
**Pull request by christianmemije**
## Summary

WIP - Coach Tools' Actions & Store

### CODE REVIEW

File: kolibri/plugins/learn/assets/src/actions.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ const router = require('kolibri.coreVue.router');
 const coreActions = require('kolibri.coreVue.vuex.actions');
 const ConditionalPromise = require('kolibri.lib.conditionalPromise');
 const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageCheckGenerator;
+const getDefaultChannelId = require('kolibri.coreVue.vuex.getters').getDefaultChannelId;
```
Comment (indirectlylit) [inline kolibri/plugins/learn/assets/src/actions.js:12]: While we're at it, we can probably move all the channel state to the core state. (e.g. the channel list and current channel ID).

thoughts?

File: kolibri/plugins/coach_tools/assets/src/actions.js [hunk 2]
```diff
@@ -14,82 +24,121 @@ function redirectToDefaultReports(store, params) {
...
 
+    error => {
+      coreActions.handleError(store, error);
+    }
+  );
+}
 
 function showReports(store, params) {
-  store.dispatch('CORE_SET_PAGE_LOADING', true);
+  store.dispatch('CORE_SET_PAGE_LOADING', true); // does this even work since I didn't implement it?
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/actions.js:71]: The `core-base` component is supposed to use this variable to hide the main `content` slot when `loading === true`

File: kolibri/plugins/learn/assets/src/actions.js (hunk 1 above)
Comment (christianmemije) [reply kolibri/plugins/learn/assets/src/actions.js:12]: Sounds very reasonable.

File: kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue [hunk 3]
```diff
@@ -160,7 +160,7 @@
 
   .summary-section
     padding: 20px
-    background-color: #bfbfbf
+    background-color: white
```
Comment (indirectlylit) [inline kolibri/plugins/coach_tools/assets/src/vue/reports/summary-section/index.vue:163]: `$core-bg-light` ;-)

File: kolibri/core/assets/src/vue/content-icon/content-icons/video.svg [hunk 4]
```diff
@@ -1,4 +1,4 @@
 <svg fill="#000000" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
-    <path d="M18 4l2 4h-3l-2-4h-2l2 4h-3l-2-4H8l2 4H7L5 4H4c-1.1 0-1.99.9-1.99 2L2 18c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V4h-4z"/>
-    <path d="M0 0h24v24H0z" fill="none"/>
+    <path d="M0 0h24v24H0V0z" fill="none"/>
+    <path d="M21 3H3c-1.11 0-2 .89-2 2v12c0 1.1.89 2 2 2h5v2h8v-2h5c1.1 0 1.99-.9 1.99-2L23 5c0-1.11-.9-2-2-2zm0 14H3V5h18v12zm-5-6l-7 4V7z"/>
```
Comment (indirectlylit) [inline kolibri/core/assets/src/vue/content-icon/content-icons/video.svg:3]: me gusta

**Pull request merged**